app/src/main/java/de/westnordost/streetcomplete/quests/sauna_availability/AddSaunaAvailability.kt

```diff
@@ -0,0 +1,33 @@
+package de.westnordost.streetcomplete.quests.sauna_availability
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddSaunaAvailability : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        nodes, ways with
+        (
+          leisure ~ fitness_centre|resort|sports_centre|water_park
```

Comment:
You want to ask for all (including school) sport centres?
IMO to much.
```suggestion
          leisure ~ fitness_centre|resort|water_park
```

Reply:
@HolgerJeromin The [Wiki](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dsports_centre) says that a sauna can be present in a sports_centre. How to differentiate?

> A sports centre is a distinct facility where sports take place within an enclosed area. It can be a building (indoor sports centre), just outside (outdoor sports centre) or contain indoor and outdoor sports features mixed together.
> 
> Facilities within a sports centre may include [swimming pools](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dswimming_pool), spa, solarium, [sauna](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dsauna) and/or steam room, [sports hall](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dsports_hall), squash courts, [fitness suite](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dfitness_centre), aerobics studios, outdoor grass and/or artificial [pitches](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dpitch) for football (soccer), hockey etc and also any associated cafeteria, bars and other facilities.

Reply:
Yes. You are correct. This **can** be included. 
But still, including them would IMO violate the no-spam rule.
13 objects around me. School sport buildings, outdoor soccer / tennis areas, one kid swimming facility.
I do not even know a `sport_centre` near me where there is a sauna included (but not mapped already)...

Reply:
Hm yeah, but it's still SCEE and you can always keep the quest disabled.

Reply:
Removed `resort|sports_centre|water_park`